Breeze/src/Breeze.Daemon/Program.cs

```diff
@@ -24,20 +24,31 @@ namespace Breeze.Daemon
 {
     public class Program
     {
+        private const string DefaultBitcoinUri = "http://localhost:5000";
```

Comment:
I don't like this so much, this is network specific so it should probably go in to the Network class when we bring it over the to fullnode